sdk/appconfiguration/app-configuration/src/index.ts

```diff
@@ -104,12 +110,16 @@ export class AppConfigurationClient {
     const regexMatch = uriOrConnectionString.match(ConnectionStringRegex);
     if (regexMatch) {
       const credential = new AppConfigCredential(regexMatch[2], regexMatch[3]);
-      this.client = new ConfigurationClient(credential, {
+      this.client = new AppConfiguration(credential, apiVersion, {
```

Comment:
Do the options you're passing need to be exposed on the user-facing constructor as well?

Reply:
I don't think we could support callers arbitrarily changing the API version since we have to statically generate the models. Our version of our client is essentially "locked" to when I generated them from the swagger file.

Reply:
Oh sorry, I meant the objects bag where you have proxy settings commented out. Do those also need to be exposed?

Reply:
Created issue to track it (not a huge change but this PR has had enough activity :))